examples/cndpfwd/parse-args.c

```diff
@@ -76,6 +76,15 @@ process_callback(jcfg_info_t *j __cne_unused, void *_obj, void *arg, int idx)
                 if (f->xdp_uds == NULL)
                     CNE_ERR_RET("UDS handshake failed\n");
             }
+        } else if (!strncmp(obj.opt->name, FIB_RULES_TAG, nlen)) {
+            if (obj.opt->val.type == ARRAY_OPT_TYPE) {
+                f->fib_rules = malloc(obj.opt->val.array_sz * sizeof(char *));
```

Comment:
We normally use calloc() to make sure the memory is zero. May not matter much here, but just to be safe.